api/src/main/java/org/pmiops/workbench/db/model/DataDictionaryEntry.java

```diff
@@ -133,4 +136,40 @@ public void setTransformedByRegisteredTierPrivacyMethods(
       Boolean transformedByRegisteredTierPrivacyMethods) {
     this.transformedByRegisteredTierPrivacyMethods = transformedByRegisteredTierPrivacyMethods;
   }
+
+  @Override
+  public boolean equals(Object o) {
```

Comment:
I would've thought that one of our frameworks could define this method for us. This looks good, but how do we remember to update it if we add a new field?

Reply:
We don't 🙃. I think this is a +1 reason to adopt some tooling like Kotlin or Immutables.

Reply:
Yeah, the savings in developer time and bug opportunities are huge.